back/src/Services/SocketManager.ts

```diff
@@ -384,22 +384,23 @@ export class SocketManager {
             // ToDo use const for https://webexapis.com/v1
             // ToDo make sure using inProgress is save enough (https://developer.webex.com/docs/api/v1/meetings/list-meetings)
             const params = `meetingType=meeting&state=inProgress&siteUrl=${WEBEX_SITE_URL}`;
-            const res = await Axios.get(`https://webexapis.com/v1/meetings?integrationTag=${roomId}&${params}`, {
-                headers: {
-                    "Content-Type": "application/json",
-                    Authorization: `Bearer ${accessToken}`,
-                },
-            });
+            const res = await Axios.get(
+                `https://webexapis.com/v1/meetings?integrationTag=${encodeURIComponent(roomId)}&${params}`,
+                {
+                    headers: {
+                        "Content-Type": "application/json",
+                        Authorization: `Bearer ${accessToken}`,
+                    },
+                }
+            );
             // ToDo add error handling
             console.log("[Back] Looking up meeting, got: ", res.data);
             const legalMeets = res?.data?.items;
             console.log("[Back] Legal meetings to choose from: ", legalMeets);
             let meetingId = legalMeets && legalMeets[0]?.id;
-            // todo get meetingLink by meetingId #11
-            let meetingLink;
+            let meetingLink = legalMeets && legalMeets[0]?.sipAddress;
             if (!meetingId) {
```

Comment:
does it make more sense to revert here like it was before `if(!meetingLink)`. Because meetingLink is something that is really important. What do you think?

Reply:
Can do! I was working under the assumption that if meetingID is not there, there's no way that meetingLink can be there, but I guess it would make sense to be sure!